evalml/pipelines/classification_pipeline.py

```diff
@@ -122,17 +123,19 @@ def score(self, X, y, objectives):
         y = _convert_woodwork_types_wrapper(y.to_series())
         objectives = [get_objective(o, return_instance=True) for o in objectives]
         y = self._encode_targets(y)
-        y_predicted, y_predicted_proba = self._compute_predictions(X, objectives)
-
+        y_predicted, y_predicted_proba = self._compute_predictions(X, y, objectives)
+        if y_predicted is not None:
```

Comment:
Nope! I was thinking about this too, but ultimately decided to stick with this. The difference, I think, is what we're expecting from the user / how the user will use it.

If we have _convert_woodwork_types_wrapper(None) return None, we're allowing the user to more freely use this and then having to deal with checking for None later, where None values don't make sense (ex: scoring).

If we disallow _convert_woodwork_types_wrapper(None), then the user needs to more conscious about their input and ensure that its not None. They shouldn't call _convert_woodwork_types_wrapper if their input is None. While updating and keeping this requirement, I caught a few instances where we were accidentally passing in None or forgetting to pass an input, and disallowing None caught it, which is why I ultimately stuck with this but open to more thoughts, or you think this would lead to a better user experience! 😁

Reply:
Sounds good!